### ISSUE TITLE
Table cell should be aligned right in a11y example

### ISSUE THREAD

**Issue by sukhrajghuman** [govau/design-system-site#599]
The population cell data should be aligned right. 

![image](https://user-images.githubusercontent.com/20184809/59395703-9daf5580-8dc8-11e9-978a-3f45f7adf3ff.png)

https://designsystem.gov.au/components/table/accessibility/
